decoration.sk

```diff
@@ -155,6 +155,8 @@ decoratives after flattening:
 	{wood type} sapling¦s = -sapling
 	{wood type} lea(f|ves) = -leaves
 
+	[any] lea(f|ves) = oak leaves, birch leaves, jungle leaves, spruce leaves, dark oak leaves, acacia leaves
+
```

Comment:
Good idea, but if you're doing `any leaves` for 1.13's aliases then it'd be good to add them further up in the file for the `1.12.2 or older` section as well.

Reply:
Will do